### PR TITLE
List format: Add 'class' parameter 

### LINKED ISSUE #3136: List format: Add class parameter
Follow-up to #2488, #3135 

To further facilitate styling of results the/a `|class=` parameter to #ask/#show needs to be implemented.

### PR DESCRIPTION
**Pull request by s7eph4n**
This PR is made in reference to: #3136 

This PR addresses or contains:
- List format: Add 'class' parameter 

This PR includes:
- [x] Tests (unit/integration)
- [x] CI build passed

Fixes #3136

### CODE REVIEW

**Comment by s7eph4n** [comment]
Small, self-contained patch. Self-merging.

**Comment by JeroenDeDauw** [comment]
Please do not self-merge non-trivial changes to code, especially after having it up for only 1h.

File: src/Query/ResultPrinters/ListResultPrinter/ListResultBuilder.php [hunk 1]
```diff
@@ -31,19 +31,19 @@ class ListResultBuilder {
 		'list' => [
 			'row-open-tag' => '<span class="smw-row">',
 			'row-close-tag' => '</span>',
-			'result-open-tag' => '<span class="smw-format list-format">',
+			'result-open-tag' => '<span class="smw-format list-format $CLASS$">',
```
Comment (JeroenDeDauw) [inline src/Query/ResultPrinters/ListResultPrinter/ListResultBuilder.php]: This looks like it is an XSS vulnerability

**Comment by s7eph4n** [comment]
Indeed. Will fix.

**Pull request merged**